### PR TITLE
[CP-3259] Wrong command to run project

### PR DESCRIPTION
**Pull request by katarzynastegienko**
JIRA Reference: [CP-3259]

### :memo: Description ️

- Wrong command to run project in mudita-center/quickstart.md 

### :muscle: Checklist before requesting a review - nice to have

- 

### :exclamation: Checklist before merging a pull request

- change went through the QA process
- translations are updated in dedicated application
- [CHANGELOG.md](./CHANGELOG.md) is updated

[CP-3259]: https://appnroll.atlassian.net/browse/CP-3259?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: quickstart.md [hunk 1]
```diff
@@ -47,13 +47,13 @@ To ensure that files will be excluded from commits you can use the following git
...
 ```bash
-npm run develop
+npm run app:develop
 ```
 
 This will start the application with hot-reload so you can instantly start developing it. You can also enable logs by executing:
 
 ```bash
-npm run develop -- --stream
+npm run app:develop -- --stream
```
Comment (dkarski) [inline quickstart.md:56]: Thanks for updating the documentation. Some commands are outdated. Now, npm run app
is sufficient without -- --stream.
```suggestion
```

Please remove this entire section 🙏

**Review by dkarski — COMMENTED** [review]
Thanks for this change. It's a good start to ensure up-to-date content and review it again, as there might be other areas needing updates.

File: quickstart.md (hunk 1 above)
Comment (katarzynastegienko) [reply quickstart.md:56]: Should I delete this?
```bash
lerna run --stream --scope @mudita/mudita-center-app dev:start:renderer
lerna run --stream --scope @mudita/mudita-center-app dev:start:main
```

Comment (dkarski) [reply quickstart.md:56]: yes, you can, thx

**Review by dkarski — APPROVED** [review]

**Pull request merged**